mail_activity_board/views/mail_activity_view.xml

```diff
@@ -0,0 +1,216 @@
+                <field name="date_deadline"/>
+                <field name="state"/>
+                <field name="icon"/>
+                <field name="activity_type_id"/>
+                <field name="activity_category"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div t-attf-class="#{kanban_color(record.state)} oe_kanban_content oe_kanban_global_click">
+                            <div class="o_kanban_record_bottom">
+                                <div class="oe_kanban_bottom_left">
+                                    <span t-attf-class="fa #{record.icon}" />
+                                    <span t-attf-class="o_activity_color_#{state}"><field name="activity_category" /> </span>
+                                </div>
+                                <div class="oe_kanban_bottom_right">
+                                    <em>due on: </em><span t-attf-class="o_activity_color_#{state}"><field name="date_deadline" /> </span>
+                                    <strong>
+                                        <t t-esc="activity_type_id" />
+                                    </strong>
+                                </div>
+                            </div>
+                            <strong><span t-attf-class="o_activity_date o_activity_color_#{record.state}"><t t-esc="record.label_delay" /></span></strong>
+                            <div>
+                                <strong class="o_kanban_record_subtitle"><field name="res_name"/></strong>
+                            </div>
+                            <div>
+                                <span class="o_kanban_record_subtitle"><field name="user_id"/></span>
+                            </div>
+                            <field name="calendar_event_id_start"/>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+         </field>
+    </record>
+
+
+    <!-- SEARCH VIEW -->
+    <record id="mail_activity_view_search" model="ir.ui.view">
+        <field name="name">search.activity.view.calendar.inherit.activities_board</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_search"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr='//field[@name="res_model_id"]' position='before'>
+                <field name="user_id"/>
+                <field name="res_name" string="Origin (opportunity/customer)"/>
+            </xpath>
+
+            <xpath expr='//filter[@name="activities_my"]' position='after'>
+                <filter string="Act. next month" name="activities_month"
+                        domain="[('date_deadline', '&lt;', (context_today()+datetime.timedelta(days=30)).strftime('%Y-%m-%d'))]"
+                        help="Show activities which date deadline is before than 1 month."/>
```

Comment:
Show activities scheduled for next month.

Reply:
I've already corrected.